core/templates/domain/exploration/ParamTypeObjectFactory.ts

```diff
@@ -96,7 +96,7 @@ export class ParamTypeObjectFactory {
   // Type registration.
 
   /** @type {Object.<String, ParamType>} */
-  registry = {
+  registry: {[key: string]: ParamType}= {
```

Comment:
Prefer creating an interface if it does not exist and use that here.

Reply:
done